### ISSUE TITLE
如何获取登录的用户头像

### ISSUE THREAD

**Issue by davieds** [youfou/wxpy#108]
使用获取头像的方法 
bot.self.get_avatar('./%s'%bot.self.name)
提示 ：TypeError: expected `Friend`, `Group` or `Member`, got`<class 'wxpy.api.chats.user.User'>`
这是只能获取 Friend Group Member 类型的吗？

**Comment by honwenle** [comment]
yes , only Friend Group Member , but you can use `is_friend` return a Friend object , then you can use `get_avatar`
``` py
bot.self.is_friend.get_avatar('myself_avatar.png')
```
